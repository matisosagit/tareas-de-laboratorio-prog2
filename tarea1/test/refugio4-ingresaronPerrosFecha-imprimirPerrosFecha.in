crearRefugio

# se crean 5 perros para ingresarlos al refugio

crearFecha 28/2/2020
crearPerro 1 Scooby 5 8 Do 
agregarEnRefugio

crearFecha 15/6/2020
crearPerro 2 Sirius 6 9 Black
agregarEnRefugio

crearFecha 15/6/2020
crearPerro 3 Remus 6 8 Lupin
agregarEnRefugio

crearFecha 8/11/2019
crearPerro 4 Beto 15 7 Beige beteado, grande 
agregarEnRefugio

crearFecha 3/9/2022
crearPerro 5 Rocky 5 5 Un bulldog ingles con caracter tranquilo, le encanta dormir 
agregarEnRefugio

# No hay perros fecha
ingresaronPerrosFechaRefugio 15/7/2022

# Hay 2 perros fecha
ingresaronPerrosFechaRefugio 15/6/2020

liberarRefugio

Fin