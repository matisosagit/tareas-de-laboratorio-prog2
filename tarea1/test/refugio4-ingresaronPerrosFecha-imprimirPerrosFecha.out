1>2>#  se crean 5 perros para ingresarlos al refugio.
3>4>5>6>7>8>9>10>11>12>13>14>15>16>17>18>#  No hay perros fecha.
19>No se encontraron perros ingresados en la fecha determinada.
20>#  Hay 2 perros fecha.
21>Perro 2
Nombre: Sirius
Edad: 6
Descripcion:  Black
Fecha de ingreso: 15/6/2020
Vitalidad: 9
Perro 3
Nombre: Remus
Edad: 6
Descripcion:  Lupin
Fecha de ingreso: 15/6/2020
Vitalidad: 8
22>23>Fin.
